src/clds_sorted_list.c

```diff
@@ -1153,7 +1157,7 @@ CLDS_SORTED_LIST_ITEM* clds_sorted_list_find_key(CLDS_SORTED_LIST_HANDLE clds_so
                     else
                     {
                         // now make sure the item has not changed
-                        if (InterlockedCompareExchangePointer((volatile PVOID*)current_item_address, (PVOID)current_item, (PVOID)current_item) != (PVOID)((uintptr_t)current_item & ~0x1))
+                        if (InterlockedCompareExchangePointer((volatile PVOID*)current_item_address, (PVOID)current_item, (PVOID)current_item) != (PVOID)current_item)
```

Comment:
>current_item [](start = 108, length = 12)

so this will now fail (or go down the first branch) if the delete bit is set?

Reply:
Well, yes. I'll look at optimizing this some other time.